src/logic/lint.ts

```diff
@@ -262,6 +266,19 @@ async function prepareLintedProfile(
     ast = await fetchProfileAST(profile.id, profile.version);
   }
 
+  //Validate examples
+  if (ast) {
+    const examplesValidationResult = validateExamples(
+      ast,
+      getProfileOutput(ast)
```

Comment:
I propose to store this into variable as you use it in validation of map as well.